lisa/sut_orchestrator/libvirt/platform.py

```diff
@@ -594,8 +594,10 @@ def _create_node(
 
         # Create libvirt domain (i.e. VM).
         xml = self._create_node_domain_xml(environment, log, node)
+        log.debug(f"Domain xml for {node_context.vm_name} - {xml}")
         node_context.domain = self.libvirt_conn.defineXML(xml)
 
+        log.debug(f"Creating domain - {node_context.vm_name}")
```

Comment:
Only really libvirt uses the term "domain". I would say "libvirt domain" or "libvirt VM".